src/conv/invokers/mlir_impl_gemm.cpp

```diff
@@ -48,6 +48,15 @@ struct MlirConvArgs
     StridedMemRef5D output;
 };
 
+// Note: Below macros are required for opencl backend only because
+// opencl backend requires to invoke clSetKernelArg on every kernel
+// arguments whereas hip can implicitly linearize the struct to
+// kernel arguments
+#if MIOPEN_BACKEND_OPENCL
+#define EXPAND_ARRAY_5(x) x[0], x[1], x[2], x[3], x[4]
```

Comment:
```suggestion
#define EXPAND_ARRAY_5(x) ((x)[0]), ((x)[1]), ((x)[2]), ((x)[3]), ((x)[4])
```
macro args must be enclosed into parentheses

Reply:
Done in 70f5570